docs/user-guide/styling-your-docs.md

```diff
@@ -28,6 +28,15 @@ The default theme, which was built as a custom [Bootstrap] theme, supports most
 every feature of MkDocs. It only officially supports two levels in the
 navigation (see #1107).
 
+!!! Warning
+   As of version 1.1 Bootstrap has been updated from 3.0.3 to 4.1.3. After this update 2 filenames were changed to avoid confusion. 
+   
+   Currently there are 2 themes which make use of inheritence from mkdocks [MkDocs Bootstrap theme](https://github.com/mkdocs/mkdocs-bootstrap) and [MkDocs Bootswatch theme](https://github.com/mkdocs/mkdocs-bootswatch),if you are using either of these and are having issues with bootstrap you may need to change the filenames paths to these files as described below.
+
+   Filename Updates:
+   css/bootstrap-custom.min.css => css/bootstrap.min.css
+   js/bootstrap-3.0.3.min.js => js/bootstrap.min.js
+
```

Comment:
Again, this information should not be included here but in the release notes and directed at theme authors, not users.